Dockerfile

```
@@ -4,7 +4,8 @@ LABEL "repository"="https://github.com/g4s8/pdd-action"
 LABEL "maintainer"="Kirill Che."
 
 # install pdd
-RUN apk add --update --no-cache ruby && \
+RUN apk add --update --no-cache xz-libs && \
+  apk add --update --no-cache ruby && \
```

You can just specify multiple targets for `add` in one command:
```suggestion
RUN apk add --update --no-cache ruby xz-libs && \
```